src/test/java/code401challenges/linkedlist/LinkedListTest.java

```diff
@@ -0,0 +1,102 @@
+package code401challenges.linkedlist;
+
+import org.checkerframework.common.value.qual.StaticallyExecutable;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+    @Test
+    public void linkedListTest() {
+        LinkedList test = new LinkedList();
+        assertNull(test.head);
+    }
+
+
+
+//    2.Can properly insert into the linked list
+    @Test
+    public void prependTest() {
+        LinkedList test = new LinkedList();
+        test.prepend(7);
+        int actual = test.head.value;
+//        check if head is pointing to node with value of 7
+       assertEquals(7, actual);
+    }
+
+
+
+//    3.The head property will properly point to the first node in the linked list
+    @Test
+    public void 
+
+
+
+//    4.Can properly insert multiple nodes into the linked list
+    @Test
+    public void propPrepend() {
+        LinkedList test = new LinkedList();
+        test.prepend(3);
+        test.prepend(5);
+        test.prepend(7);
+        test.prepend(9);
+        test.prepend(10);
+        Node current = test.head;
+        int nodeCount=0;
+        for(int i=0;i<5;i++){
+            nodeCount++;
+            current = current.next;
+        }
+        assertEquals( 5, nodeCount);

```

Comment:
I see what you're doing here, but it's a little confusing... at least add a message to this test. Imagine you are writing the code and tests for a company, then it's 5 years later, you don't work there anymore, and a new developer is trying to tweak some stuff in your code, but then starts failing tests and wants to know why.